### PR TITLE
EL-1342 tidy up content on applicant page

### PR DESCRIPTION
**Pull request by starswan**
[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-1342)

## What changed and why

Change content on applicant page to have summary reveals rather than large amounts of static text 

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -924,6 +925,7 @@ en:
         legend: What is your client's employment status?
         hint: "You cannot use this service if your client, or their partner, is self-employed."
       passporting:
+        definition: Help with passporting benefits
         legend: Does your client receive a passporting benefit?
         partner_hint: Also select yes if your client is named on their partner’s passporting benefit claim.
```
Comment (MazOneTwoOne) [inline config/locales/en.yml:930]: @starswan a question for UCD, but do we still need this text as it wasn't included in the new design? 

> Also select yes if your client is named on their partner’s passporting benefit claim.

**Review by MazOneTwoOne — APPROVED** [review]
Looks good and approving. Tested out Mac Voice Over and fixes that finding. 

My only other question is for UCD - as a bit of text might have been accidentally omitted on the design.

File: config/locales/en.yml (hunk 1 above)
Comment (MazOneTwoOne) [reply config/locales/en.yml:930]: @starswan had confirmation from UCD that this line now goes 👍🏽

Comment (starswan) [reply config/locales/en.yml]: done

**Pull request merged**